Player/Player.gd

```diff
@@ -0,0 +1,26 @@
+extends CharacterBody2D
+
+
+var speed = 200
```

Comment:
This should be a `const` https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_basics.html